lib/ansible/cli/galaxy.py

```diff
@@ -1122,7 +1122,7 @@ def _execute_install_role(self, requirements):
             try:
                 installed = role.install()
             except AnsibleError as e:
-                display.warning(u"- %s was NOT installed successfully: %s " % (role.name, to_text(e)))
+                display.warning(u"- %s %s was NOT installed successfully: %s " % (role.name, role.version, to_text(e)))
```

Comment:
Not all roles have a version. We should handle that by either not including `role.version` in the message or setting it to `*`.

Reply:
Sure. I have made that change. Thanks for the @samdoran